lib/vicadmin/validate.go

```diff
@@ -130,41 +130,41 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
 		v.FirewallIssues = template.HTML("")
 		v.LicenseStatus = BadStatus
 		v.LicenseIssues = template.HTML("")
-	} else {
+		return nil
```

Comment:
This hunk looks like a bunch of changes but really all I did was add `return nil` to the end of the `if` statement and outdent the `else` block & remove the `else` keyword.